org.skytemple.SkyTemple.yml

```diff
@@ -0,0 +1,136 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
```

Comment:
do you really need the GNOME runtime?

Reply:
Hm, not entirely sure, this is a Gtk 3 application and even though it doesn't have a lot of specific features it's still primarily targeted for the GNOME desktop. 

I'll test test with the freedesktop Sdk/Platform tomorrow.

Reply:
Actually now that I think about it, I do think I need it for things like out of the box Gtk libs, PyGObject etc.

Reply:
Gtk3 is in the Freedesktop runtime. However not PyGObject

Reply:
Would you suggest I switch to freedesktop and install PyGObject manually or stay on GNOME?
While I don't think I'll add any GNOME specific features in the near future (especially since this app is also supposed to run on macOS and Windows without much hassle), I'll probably in the distance future when I switch to Gtk4 also switch to using libadwaita and whatever that entails.